### DISCUSSION TITLE
How do I apply a css wrapper to a relationship field selector when the field has subfields?

### DISCUSSION THREAD

**Discussion by kevin-r-j** [Laravel-Backpack/community-forum#830] [Q&A (Help)] [answered]
I have a belongsToMany(n-n) relationship field with subfields inside it's own tab panel. I want to set the col width of the relationship select element but the wrapper applies to the whole relationship card. I can only declare input wrappers to the subfields (Reason has `col-md-8`). How can I set the select form group to `col-md-4` so the two elements are on a single row?

Saving, updating, and creating new items all works as expected.

![image](https://github.com/Laravel-Backpack/community-forum/assets/45818465/1ee8bf97-103b-4044-8e59-da2bf6bf19f9)

**Comment by jcastroa87** [comment, marked as answer]
Hello @kevin-r-j 

Can you share the code, so we can help you.

Here is an example about handle the wrapper on pivot

```
[
'name'    => 'dummyproducts',
'label'   => 'BelongsToMany (n-n) <small>+ subfields for pivot table</small>'.backpack_new_badge(),
'wrapper' => [
    'class' => 'form-group col-md-4',
],
'pivotSelect' => [
    'wrapper' => [
        'class' => 'form-group col-md-6',
    ],
],
'subfields' => [
    [
        'name'    => 'notes',
        'type'    => 'text',
        'wrapper' => [
            'class' => 'form-group col-md-6',
        ],
    ],
],
'tab'   => 'Relationship',
],
```

Cheers.

**Reply by kevin-r-j** [reply]
This is exactly it and I hadn't spotted the pivotSelect section in the docs right below where I was looking. 
Thanks for saving my sanity and any others who come along!
